projects/novo-elements/src/elements/tabbed-group-picker/TabbedGroupPicker.html

```diff
@@ -48,7 +48,7 @@
           #tabbedGroupPickerVirtualScrollViewport>
           <novo-option
             *cdkVirtualFor="let item of displayTab.data"
-            [attr.data-automation-id]="item[displayTab.labelField]"
```

Comment:
technically this is a breaking change, but i don't think anyone uses these automation-ids besides us so i'm okay with it. if we want to be 100% safe we can leave this as-is and add the valueField as a `data-automation-value`, but that will probably just be more confusing for people in the future.